### ISSUE TITLE
Support older versions of e2fsprogs (or custom filesystem creation flags)?

### ISSUE THREAD

**Issue by brunotm** [vmware/docker-volume-vsphere#629]
Older versions of e2fsprogs (mkfs.ext2/3/4) asks for confirmation when creating a filesystem on a entire device and not a partition (eg. /dev/sdb) and make the filesystem creation to fail.

 This was due to a broken check which was removed in [06/2015](http://git.kernel.org/cgit/fs/ext2/e2fsprogs.git/commit/?id=97e9b970769b1afe462f82390cb5876cf037b903).

Should support for these versions be added?
If so, add them through logic in fs.Mkfs() or through a fs-config.json configuration file, allowing users to customize filesystem creation for any filesystem?

Eg.

```
{
    "ext4": {
        "flags": ["-F", "-m", "1"]
    },
    "btrfs": {
        "flags": ["-d", "single"]
    },
    "xfs": {
        "flags": ["-l", "lazy-count=1"]
    }
}
```

**Comment by govint** [comment]
IOW if the docker volume create should include an option (along with the
rest that's supported today) of what FS to create the command to run and
the client simply runs that command. Allowing any FS to be used to format
the disk and for those apps that intend to use a raw disk a "no-format"
option as well.

**Comment by brunotm** [comment]
@govint @msterin 

This is also needed for #612, unless there is no need to support linux distributions with older version of e2fsprogs. - we know that the VM2 used in the CI (ubuntu 14.04) has an older version.

**Comment by msterin** [comment]
Resolved by #631
